homeassistant/components/swiss_public_transport/config_flow.py

```diff
@@ -30,7 +30,7 @@
 class SwissPublicTransportConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     """Swiss public transport config flow."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
This can be migrated backward compatible, in those cases, we should bump a minor version.

See also: <https://developers.home-assistant.io/blog/2023/12/18/config-entry-minor-version>

Reply:
I was thinking about this, but making it a minor version would blow up when reverted to when someone has more than 1 entry as every entry would register their sensor with departure_None as unique id

Reply:
Not sure currently, what is needed here, Minor or Major. If I should switch to Minor, just let me know :)

Reply:
@joostlek It would bring back the previous state yes. Which maybe not ideal, but the data itself is not backward-incompatible.

Besides, we can't ship majors in patch releases.